package.json

```diff
@@ -5,7 +5,7 @@
     "analyze": "webpack --mode=production --env.analyze=true",
     "lint": "eslint src --ext js,ts,tsx",
     "format": "prettier --write './**'",
-    "test": "jest",
+    "test": "cross-env BABEL_ENV=test jest",
```

Comment:
By setting the BABEL_ENV, we can tell babel to behave differently during tests versus during normal compilation.